src/matrix.jl

```diff
@@ -172,13 +172,53 @@ We exploit that
 to avoid taking the absolute (``|·|``).
 """
 function LinearAlgebra.opnorm(A::IntervalMatrix, p::Real=Inf)
-    if p == Inf || p == 1
-        return LinearAlgebra.opnorm(max.((-).(inf(A)), sup(A)), p)
+    if p == Inf
+        return _opnorm_inf(A)
+    elseif p == 1
+        return _opnorm_1(A)
     else
         error("the interval matrix norm for this value of p=$p is not implemented")
     end
 end
 
+# The operator norm in the infinity norm corresponds to the
+# maximum absolute value row-sum. Julia has column-major
+# ordering so for efficiency we iterate over the transpose of A.
+function _opnorm_inf(A::IntervalMatrix{N}) where {N}
+    m, n = size(A)
+    res = zero(N)
+    At = transpose(A)
+    @inbounds @simd for j in 1:m
+        acc = zero(N)
+        for i in 1:n
+            x = At[i, j]
+            acc += max(abs(inf(x)), abs(sup(x)))
```

Comment:
```suggestion
            acc += max(-inf(x), sup(x))
```
This is what the documentation says and it is more efficient.

Reply:
It is not more efficient, i tried..

Reply:
I don't know why it is more efficient, with the double abs, because used in isolation, `max(-inf(x), sup(x))` is better. But note that version `_opnorm_inf_f` is much worse!

Reply:
For reference, up to dimension ~60 both versions show the same speed. For higher dimension the version with double abs becomes faster.